### ISSUE TITLE
Cannot navigate by list items in definition lists in Chromium browsers

### ISSUE THREAD

**Issue by tmthywynn8** [nvaccess/nvda#16590]
When encountering description lists in Chromium browsers, one cannot navigate by list items (*I), however NVDA still provides the list semantics and one can navigate to said list (*L*). This works as expected in Firefox 126.

### Steps to reproduce:

1. In a Chromium browser, open the following by pasting in the address bar:  
    ```
    data:text/html,<p>Description list with three terms:</p><dl><dt>Term 1</dt><dd>Description 1</dd><dt>Term 2</dt><dd>Description 2</dd><dt>Term 3</dt><dd>Description 3</dd></dl>
    ```
2. Navigate to the description list by pressing *L*.
3. navigate by list items by pressing *I*.

### Actual behavior:
```
{pressing L}: list with 6 items Term 1
{Pressing I}: no next list item
```

### Expected behavior:
```
{Tested in Mozilla Firefox Version 126.0 (64-bit)}:
{Pressing L}: list with 6 items Term 1
{Pressing I}: Term 2
```

#### NVDA installed/portable/running from source:

Installed

#### NVDA version:

2024.2beta2

#### Windows version:

Windows 10 Enterprise Version 21H2 (OS build 19044.3086)

#### Name and version of other software in use when reproducing the issue:

* Microsoft Edge for Business Version 125.0.2535.51 (Official build) (64-bit)
* Google Chrome Version 125.0.6422.77 (Official Build) (32-bit)

#### Does the issue still occur after restarting your computer?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

NVDA 2023.3 exhibits the same behavior.

#### If NVDA add-ons are disabled, is your problem still occurring?

Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

Yes

**Comment by seanbudd** [comment]
Closing as duplicate of #3858

**Comment by tmthywynn8** [comment]
Reopening because I don't think this is a duplicate of #3858. The terms are navigable in Firefox with *I* and but not with Chromium browsers. The issue that is believed to be a duplicate of relates to how terms and descriptions are navigated/announced by NVDA and the best way of presenting that semantic information. Please read my STR, expected results, and actual results again, and if you believe it to be a duplicate issue, please let me know.

**Comment by Adriani90** [comment]
Description lists are more and more used in the wild, especially when interviews are published, whereby Questions are the terms and answers are their description.
e.g. for An interview with 3 questions and 3 answers, NVDA reports in Firefox "list with 6 items", but when using the quick nav key i for list items, NVDA navigates only the 3 questions which are treated like list items.
cc: @jcsteh what is the reason behind this behavior? In my opinion NVDA should report list with 3 items since the description of a term is part of a list item itself.

**Comment by Adriani90** [comment]
@aleventhal, @dsexton should we create an Issue on the Chromium bug tracker as well? Or is there one already?
In Chromium, NVDA also reports terms and their descriptions as separate list items when entering a description list, although the description of a term is not a separate list item.

In Chromium, pressing quick navigation key i for list items doesn't recognize any of them in a description list.
cc: @SaschaCowley is this something that can be fixed in NVDA's virtual document?

**Comment by Adriani90** [comment]
In Chromium Jaws even doesn't recognize that there is a list at all. NVDA at least can say that there is a list but doesn't recognize the list items.
I've created https://issues.chromium.org/issues/376278217

**Comment by andrew-l-d** [comment]
On a page I was asked to look at today current JAWS both announced that a definition list was a definition list and navigated to <dt>s with i in (I am pretty sure) all major browsers.  I believe it is important that a definition list is reported as such.  At risk of excessive verbosity, should titles be reported?  Perhaps an option in Document Formatting.

**Comment by tmthywynn8** [comment]
I attempted to navigate through the previously supplied definition list with JAWS 2026.2510.251, Narrator from Windows 11 24H2 Enterprise, and NVDA 2025.3.2 on Google Chrome 143.0.7499.110, and here are my results when navigating by list items (*I*):

* JAWS: Navigates between the three definition terms .
* Narrator: Navigates between the three definition terms .
* NVDA: Says "no next list item"
I am not sure if JAWS or Narrator have different heuristics, but it works in two of the three Windows screen readers I have on hand to test.
